Makefile

```diff
@@ -25,6 +25,11 @@ run: build/initrd/.id .dapper
 	./.dapper -m bind build-target
 	./scripts/run
 
+iso:
+	qemu-system-x86_64 \
+		-curses -net nic -net user \
+		-m 2048M -boot d -cdrom dist/artifacts/rancheros.iso
```

Comment:
Does this need to be in this PR?  Should we instead pursue running qemu/kvm in a container, more qind style?